src/components/app-wrapper.js

```diff
@@ -128,11 +124,27 @@ const darkTheme = createTheme({
     },
 });
 
+const customDarkTheme = createTheme(darkTheme, {
```

Comment:
I would prefer this solution here : https://stackoverflow.com/a/69450044/5359613
to avoid theme customisation in darkTheme fro some, then in customDarkTheme for others
```
let defaultTheme = createTheme();
const lightTheme = createTheme(defaultTheme, {...});
const darkTheme = createTheme(defaultTheme, {...});
```

Reply:
I relied on the MUI documentation in the 'Using Theme Options to Define Other Options' section'
https://mui.com/material-ui/customization/theming/#api

Reply:
You apply theme composition here but we just want a composition between the default theme and our custom theme
* before : defaultTheme + lightTheme|darkTheme
* with your change : defaultTheme + lightTheme|darkTheme + customDarkTheme|customLightTheme
this is not necessary I think,  lightTheme|darkTheme are already our custom themes

Reply:
You cannot pass defaultTheme as a parameter to createTheme in this way, it will not work because createTheme does not expect to receive an existing theme as the first argument, but rather a configuration object or nothing at all. Therefore, one can use the defaultTheme constant directly without passing it as a parameter. This works, but you need to set the mode so that the text color adapts to the theme; otherwise, it will always be black (the default color). For example:
` const defaultTheme = createTheme({ palette: { mode: 'dark', }, });`

Reply:
Done